services_app/src/main/java/org/opendatakit/services/preferences/activities/AppPropertiesActivity.java

```diff
@@ -75,11 +80,15 @@ public PropertiesSingleton getProps() {
     return mProps;
   }
 
+  //
```

Comment:
Why is there a blank comment block?

Reply:
I did that mistakenly, I'll fix that. Thanks